src/game.h

```diff
@@ -733,7 +733,8 @@ class game
         // Data Initialization
         void init_autosave();     // Initializes autosave parameters
         void create_starting_npcs(); // Creates NPCs that start near you
-        bool place_starting_vehicle(); // try and place starting vehicle on a nearby road.
+        void place_starting_vehicle( cata::optional<vproto_id>
+                                     starting_vehicle ); // try and place starting vehicle on a nearby road.
```

Comment:
Please move the comment before the function declaration, so the declaration is not split across multiple lines.